client/src/components/pages/profile/profile-bio/index.tsx

```diff
@@ -69,7 +68,7 @@ export const ProfileBio: React.FC<IProfileBioProps> = ({
 				{avatar ? <Avatar avatar={avatar} size={57} /> : <Avatar size={57} />}
 				<h5 className={styles.name}>{`${name} ${surname}`}</h5>
 				<Rank rank={rank ? rank : 9} />
-				<Label label={score ? score : 0} color="#EC4179" />
+				<Rank honor={honor ? honor : 0} />
```

Comment:
why do you use Rank component to display honor?

Reply:
because it Rank components is done for that too. The colors there work properly then too.